wiki/ACE_contig_to_alignment.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: ACE contig to alignment
+title: Visualize an alignment from conting archived in ACE files.
```

Comment:
Typo: contig

I think visualise isn't quite the right description here, maybe leave it as it was?